src/main/java/com/blackduck/integration/detect/tool/signaturescanner/operation/CreateScanBatchOperation.java

```diff
@@ -116,5 +125,16 @@ private ScanBatch createScanBatch(
             throw new DetectUserFriendlyException(e.getMessage(), e, ExitCodeType.FAILURE_CONFIGURATION);
         }
     }
+    
+    private boolean conditionalCorrelationFilter(boolean toCheck, String toWarn) {
+        if (toCheck) {
+            if (signatureScannerOptions.isIntegratedMatchingEnabled()) {
+                logger.warn("{} is not compatible with Integrated Matching feature and will be skipped. Please re-run {} with integrated matching disabled.", toWarn, toWarn.toLowerCase());
```

Comment:
Is it called Integrated Matching as a feature, or Correlated matching with Integrated Matching as the property for it?

Reply:
Integrated Matching which is consistent with the property name.

Reply:
Match Correlation has been used in the past.

Reply:
In our release notes and in Black Duck the new feature is referred to as Correlated Scanning isn't it. (Or am I thinking of two different features?)

Reply:
That is a valid point. What about the property name - Integrated Matching Enabled?

Reply:
I hate to say it, but may be worth aligning the naming? (See what others think)

Reply:
@sig-cpotts Property has been renamed to align with Black Duck HUB.